packages/taquito/src/constants.ts

```diff
@@ -50,7 +52,7 @@ export enum DefaultLambdaAddresses {
   FLORENCENET = 'KT1KCe3YqGnudsiCWb5twbe2DH5T3EMdLpSE',
   GRANADANET = 'KT1BCun2vsA4GBQvsKAuGD5x873MfW2jsN9z',
   HANGZHOUNET = 'KT1PWtBAr1hjK9M9s9oZNZFbfzPdkkD6PSJR',
-  IDIAZABALNET = 'KT1HEnAszfBjRgUpXYfHNaHnjFbb6iZY247J'
+  ITHACANET = 'KT1HEnAszfBjRgUpXYfHNaHnjFbb6iZY247J'
```

Comment:
We need to deploy the lambda contract to ithacanet. KT1HEnAszfBjRgUpXYfHNaHnjFbb6iZY247J is deployed in idiazabalnet.

Reply:
I used the example/deploy-lambda-contract.ts to originate contract  KT1CsEGfRHWeuUQFh9LfVFLVMbm7DFBuHPPU on Ithacanet